stable/mysql/templates/NOTES.txt

```diff
@@ -34,9 +34,8 @@ To connect to your database directly from outside the K8s cluster:
     MYSQL_HOST=127.0.0.1
     MYSQL_PORT={{ default "3306" .Values.service.port }}
 
-    # Execute the following commands to route the connection:
-    export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app={{ template "mysql.fullname" . }}" -o jsonpath="{.items[0].metadata.name}")
-    kubectl port-forward $POD_NAME {{ default "3306" .Values.service.port }}:{{ default "3306" .Values.service.port }}
+    # Execute the following command to route the connection:
+    kubectl port-forward svc/{{ template "mysql.fullname" . }} {{ default "3306" .Values.service.port }}:{{ default "3306" .Values.service.port }}
```

Comment:
is the default needed, see: https://github.com/helm/charts/blob/master/stable/mysql/templates/svc.yaml - it uses .Values.service.port